projects/driver_controls/inc/can_fsm.h

```diff
@@ -3,5 +3,22 @@
 // FSM for transmitting driver input data over CAN
 
 #include "fsm.h"
+#include "input_event.h"
 
+// Device IDs for CAN output events
+typedef enum {
+  CAN_DEVICE_ID_POWER = NUM_INPUT_EVENT,
+  CAN_DEVICE_ID_PEDAL,
+  CAN_DEVICE_ID_DIRECTION_SELECTOR,
+  CAN_DEVICE_ID_TURN_SIGNAL,
+  CAN_DEVICE_ID_HAZARD_LIGHT,
+  CAN_DEVICE_ID_MECHANICAL_BRAKE,
+  CAN_DEVICE_ID_HORN,
+  CAN_DEVICE_ID_PUSH_TO_TALK,
+} CANDeviceID;
```

Comment:
If these are events you probably want to make it more obvious - it's not very obvious. Also, since this is a global event queue, I'd recommend adding a comment to `input_event.h` pointing here in case someone wants to add a new event.

Really, I'm not a fan of representing the device ID using the event ID directly. Separating events into multiple files is pretty confusing and will make it difficult to keep track of. It also means you need to include `input_event.h`, a seemingly unrelated header.